corehq/apps/userreports/fixtures.py

```diff
@@ -50,7 +54,14 @@ def __call__(self, user, version, last_sync=None):
             return []
 
         root = ElementTree.Element('fixture', attrib={'id': self.id})
-        reports_elem = ElementTree.Element('reports')
+        reports_elem = ElementTree.Element(
+            'reports',
+            attrib={
+                'last_sync': ServerTime(datetime.utcnow()).user_time(
+                    user.get_time_zone() or pytz.utc
```

Comment:
Err, except that does something different. Ideally `get_timezone_for_user` would try calling `user.get_time_zone()` first, but for now never mind